apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -56,7 +56,9 @@ const StopPageRedesign = ({
       {/* Route and Map Div */}
       <div className="d-flex">
         <div style={{ minWidth: "50%" }}>
-          <div>Route schedules & maps / Upcoming Trips PLACEHOLDER</div>
+          <h1 className="route-schedules-heading hidden-sm-down">
```

Comment:
[It's best practice to only use one `<h1>` on a page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page), which in this case will be the stop name at the top of the page. That's why my [previous comment](https://github.com/mbta/dotcom/pull/1534#discussion_r1136030522) suggested to use `<h2>` and `<h3>` here -- please fix.